src/main/java/frc/robot/legacy/OldConstants.java

```diff
@@ -1,152 +1,152 @@
-package frc.robot.legacy;
+// package frc.robot.legacy;
```

Comment:
Any reason you're not deleting this lines instead of commenting them? It should be easy to import what we need later.

Reply:
I didn't want to instantiate an instance of swervemodule because it might randomly add to CAN usage